core/nginx/conf/nginx.conf

```diff
@@ -244,6 +244,13 @@ http {
       {% endif %}
       {% endif %}
 
+      {% if API == 'true' %}
```

Comment:
I wonder if we still need to test against strings here now that we normalize in master